dm/relay/meta.go

```diff
@@ -234,6 +234,13 @@ func (lm *LocalMeta) doFlush() error {
 		return terror.ErrRelayNoCurrentUUID.Generate()
 	}
 
+	// it happens when switching to a new server, and meta is reset,
```

Comment:
this is not needed now? since we can make sure Meta file is written a right filename before reading, by your modification of `relay.go`

Reply:
good point